Automattic/jetpack
# Remove Boost Setup_Prompt banner on plugins page.

This PR removes any Boost banners from non-Jetpack wp-admin pages. In this case, more specifically it removes the Boost Setup_Prompt banner that shows on the wp-admin Plugins page after Boost plugin activation
(see screenshot).

![Markup 2023-12-22 at 12 02 38](https://github.com/Automattic/jetpack/assets/11078128/45599fb7-2466-4e0e-bac7-ea35f62d2d88)


This banner is being removed as part of the _PT: Removing banners and JITMs from non-Jetpack pages_ initiative/project: pbNhbs-9b1-p2






## Proposed changes:

- Remove the Boost Setup_Propmt banner from showing on the wp-admin Plugins page.

### Other information:


## Jetpack product discussion


PT: Removing banners and JITMs from non-Jetpack pages: pbNhbs-9b1-p2
Removing banners and JITMs from non-Jetpack pages – Implementation plan: pbNhbs-9da-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






- Apply this patch locally or using the Jetpack Beta plugin.
- Activate the Boost plugin
- Go to the wp-admin Plugins page.
- Verify you do not see the Boost Setup prompt/banner at the top of the page.
